src/openvpn/route.c

```diff
@@ -287,6 +287,38 @@ get_special_addr(const struct route_list *rl,
     return false;
 }
 
+static bool
+ipv6_get_special_addr(const struct route_ipv6_list *rl,
```

Comment:
This definitely would need some documentation. The behavior seems complicated.

Reply:
@fkooman I agree it'd be nice to add some comments regarding what the function is doing. It's a sizeable piece of code with no indication, so it would be bad to have extra text.